flatpak/net.kuribo64.melonDS.yml

```diff
@@ -11,12 +11,18 @@ finish-args:
   - "--share=network"
   - "--device=all"
   - "--filesystem=home"
+cleanup:
+  - "/include"
+  - "/share/man"
+  - "/lib/pkgconfig"
 modules:
-  - name: libpcap
+  
+  - name: libslirp
+    buildsystem: meson
     sources:
       - type: archive
-        url: http://www.tcpdump.org/release/libpcap-1.9.0.tar.gz
-        sha256: 2edb88808e5913fdaa8e9c1fcaf272e19b2485338742b5074b9fe44d68f37019
+        url: https://gitlab.freedesktop.org/slirp/libslirp/-/archive/v4.2.0/libslirp-v4.2.0.tar.gz
```

Comment:
`libslirp` has a newer version available - 4.3.1

Reply:
Newer versions fail to build because of the [`version : run_command`](https://gitlab.freedesktop.org/slirp/libslirp/-/blob/v4.3.1/meson.build#L2) in the meson.build file. I tried to replace it using `sed` to get around it, but it didn't work.

Reply:
Solved it by creating a patch file, it now works with the latest version of libslirp.

Reply:
Could you try using this - https://elmarco.fedorapeople.org/libslirp-4.3.1.tar.xz - without the patch?
For some reason the official archive fails for me too but this one works.

Reply:
Another option is to fetch it from git with a tag for the version instead, that solves the problem of the version being undefined when building.

Reply:
Yeah, it works. This is probably because the .tarball-version file isn't generated for some reason, but is already present in the alternative archive.

Reply:
@nadiaholmquist thanks! This is a much better solution.